Point_set_processing_3/include/CGAL/structure_point_set.h

```diff
@@ -185,7 +185,7 @@ class Point_set_with_structure
         \cgalParamDescription{a property map associating the index of a point in the input range
                               to the index of plane (`-1` if the point is not assigned to a plane)}
         \cgalParamType{a class model of `ReadablePropertyMap` with `std::size_t` as key type and `int` as value type}
-        \cgalParamDefault{unused}
+        \cgalParamDefault{There is no default, this parameters is mandatory}
```

Comment:
```suggestion
        \cgalParamDefault{There is no default, this parameter is mandatory.}
```

Reply:
That's something I indeed discussed with Laurent but I realize it did not worth handling the backward compatibility issue.